modules/vault-cluster/variables.tf

```diff
@@ -29,6 +29,10 @@ variable "allowed_inbound_security_group_ids" {
   type        = "list"
 }
 
+variable "allowed_inbound_security_group_count" {
+  description = "A count of allowed_inbound_security_group_ids used internally due to terraform limitation"
```

Comment:
Let's add a more detailed explanation here so users don't get confused. How about something like:

_The number of entries in var.allowed_inbound_security_group_ids. Ideally, this value could be computed dynamically, but we pass this variable to a Terraform resource's 'count' property and Terraform requires that 'count' be computed with literals or data sources only._